spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -980,6 +980,33 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("Chr with null character") {
+    // test compatibility with Spark, spark supports chr(0)
+    Seq(false, true).foreach { dictionary =>
+      withSQLConf(
+        "parquet.enable.dictionary" -> dictionary.toString,
+        SQLConf.ANSI_ENABLED.key -> "true",
+        CometConf.COMET_ANSI_MODE_ENABLED.key -> "true",
```

Comment:
Wondering if ANSI is related to this change...

Reply:
not required, removed ANSI configs.